core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -251,6 +251,15 @@
       <trans-unit id="ERR.moduleWithoutColumn">
         <source>Please assign all modules to a column!</source>
       </trans-unit>
+      <trans-unit id="ERR.invalidHttpUrl">
+        <source>Please enter a valid URL starting with either http:// or https://!</source>
+      </trans-unit>
+      <trans-unit id="ERR.customRgxp">
+        <source>Invalid format.</source>
```

Comment:
👍 May be _Invalid regular expression format._ instead?

Reply:
Users do not need to know about the internal use of regex, do they?

Reply:
Oh right, I mixed it up with the `ERR.customRgxp` error message. Not sure how to phrase `ERR.customRgxp` differently then, as the system has no semantic information about what the format should be.

Reply:
May be it should be possible to define the error message within the form generator itself? As an additional input for the form widget.

Reply:
> May be it should be possible to define the error message within the form generator itself? As an additional input for the form widget.

If we go this route though, my thinking is that it would be useful to be able to define a custom error message for _all_ input fields (which we are offering in our [inspiredminds/contao-extended-form-fields](https://github.com/inspiredminds/contao-extended-form-fields) extension).

Reply:
That would be a new feature, though. We should try to get this PR merged today if we want the functionality to be in Contao 4.11, therefore why don't we merge the generic but technical error message now and add a field for a custom error message for all input fields in Contao 4.12?